README.md

```diff
@@ -1,32 +1,79 @@
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/d38b72ff-f90c-4f83-8bb5-454f92168d1d.png" alt="Roundabout"><br>Roundabout</td>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/1d4b8706-0bb6-4793-b57c-2b35eb020650.png" alt="Traffic Lights"><br>Traffic Lights</td>
+  </tr>
+</table>
+
+
+## Reward
+The reward for TorchDriveEnv is based on waypoints that are provided to the ego agent, as well as a reward for movement and smoothness. Episodes that cause infractions (causing collisions or going offroad) are terminated early, otherwise the episode is terminated after a fixed number of steps. For a full overview of the reward, refer to the paper linked below. 
+
 # Installation
 
-The basic installation of torchdriveenv uses an OpenCV renderer, which is slower but easy to install. PyTorch3D renderer can be faster, but it requires specific versions of CUDA and PyTorch, so it is best installed in Docker.
+The basic installation of torchdriveenv uses an OpenCV renderer, which is slower but easy to install. PyTorch3D renderer can be faster, but it requires specific versions of CUDA and PyTorch, so it is best installed using Docker.
+
+TorchDriveEnv comes with an example that integrates [Stable-Baselines3](https://stable-baselines3.readthedocs.io/en/master), which can be used for testing purposes.
 
 ## Opencv rendering
 
-To install the “torchdriveenv” with opencv rendering:
+To install the “torchdriveenv” with opencv rendering it suffices to simply:
 ```
-pip install torchdriveenv
+pip install torchdriveenv[baselines]
 ```
-
-To run examples:
-Set the `$IAI_API_KEY` and `$WANDB_API_KEY`
+An example can be accessed by first cloning this repository and then:
 ```
-pip install torchdriveenv[baselines]
 cd examples
 python rl_training.py
 ```
 
 ## Pytorch3d rendering
 
-To install the “torchdriveenv” with Pytorch3d rendering:
+To install torchdriveenv with Pytorch3d rendering, we recommend an installation through docker. A Dockerfile is provided and can be built using:
 ```
 docker build --target torchdriveenv-first-release -t torchdriveenv-first-release:latest .
 ```
 
-To run examples:
-Set the `$IAI_API_KEY` and `$WANDB_API_KEY`
+An example can be accessed by first cloning this repository and then:
 ```
 cd examples
 docker compose up rl-training
 ```
+## NPCs
+To use the NPC's provided through the InvertedAI API, provide a valid API key through the environment variable `IAI_API_KEY`.
+
+## Wandb
+TorchDriveEnv is integrated with Weights & Biases for experiment logging, to make use of it provide an API key through `WANDB_API_KEY`.
+
+## Running experiments
+The results reported in the paper can be reproduced by cloning this repository and running:
+
+``` 
+python rl_training.py --config_file=env_configs/multi_agent/sac_training.yml
```

Comment:
The usage here might be outdated. The config files are refactored into: https://github.com/inverted-ai/torchdriveenv/blob/95fe6ff3c5e3036d0b837ac71d149c0b3d2448a1/examples/env_configs/rl_training.yml

Reply:
Wilder updated this in his last PR, so that users can run all the experiments in the paper without having to modify any files.